src/frontend/pages/_app.tsx

```diff
@@ -54,47 +51,19 @@ if (typeof window !== 'undefined') {
     apiKey: window.ENV.BUGSNAG_API_KEY,
     plugins: [new BugsnagPluginReact()],
   });
-
-  BugsnagPerformance.start({
+  const options = {
```

Comment:
You don't need a declared object for it if it's used in only one place.

If you nevertheless define it, you can set its proper type instead of any (but better not to defined it :) ):
```suggestion
  const options: BrowserConfiguration = {
```

Reply:
This was intentional because I added the `batchInactivityTimeoutMs` option, which doesn't exist on the `BrowserConfiguration` type. However, it's supported internally. To resolve the TypeScript build issue, I moved it out and typed it as `any`.